bikes/views.py

```diff
@@ -25,3 +26,8 @@ def create(self, request, *args, **kwargs):
             status=status.HTTP_201_CREATED,
             headers=headers,
         )
+
+
+class RentedBikesListAPIView(ListAPIView):
+    queryset = Bike.objects.filter(status=BikeStatus.in_service)
```

Comment:
`in_service` oznacza że jest przez jakiegoś usera wypożyczony? Pan Bródka zaznaczył, że ta nazwa jest nieintuicyjna na frontendzie. Może wypada również na backendzie zmienić nazewnictwo.

Reply:
Nazwewnictwo jest zgodne ze specyfikacją:
```State: enum BikeState {Working, InService, Blocked}```
Myślę że wystarczy jeśli zrobimy mapowanie na frontendzie (tzn co innego będzie szło z backendu, a co innego będziemy wyświetlać)

Reply:
Ale tak, domyślam się że `working` to sprawny rower nieużywany, `in_service` to używany aktywnie rower, a `blocked` to zablokowany przez techa